### DISCUSSION TITLE
Multibooking RSVP Mails when set to Approve on Payment

### DISCUSSION THREAD

**Discussion by Sp0run** [liedekef/events-made-easy#293] [General]
I have a multibooking for set up which is all working fine. The multibooking RSVP mails work great until I set it to approve on payment, then it will send a mail for each event in the booking and ignores the multibooking placeholders. 

Is there a way I can work round this? I'd rather not spam the booker with multiple emails.

Thanks,
Steve

**Comment by liedekef** [comment]
Per the documentation on multibooking mails: "Remark: multibooking mails are only sent for the initial booking, so if afterwards you mail people for approval or payment notifications, those are individual mails."

**Comment by Sp0run** [comment]
I have found a workaround for this. It causes a bit of manual work, but it saves having to spam the booker with multiple emails.

1. Enable Pending Booking Emails
2. Go to settings > Email Templates and clear out the subject of the pending email template to prevent it sending all the time
3. Create your multibooking email template and assign it in the 'Booking Pending Email' section of the RSVP tab
4. Disable auto approve on payment in the event 
5. Disable the booking paid email template for this event

When someone books onto the event they will be sent the multibooking email and you will have to manually approve the booking.

Theres a couple of problems with this method:
1. The pending email will be sent whether the user pays or not (I put a disclaimer on the mail to say the booking is pending until payment is received)
2. You have to manually approve each booking (although its quite easy to check that payment has been received)

I think they are all the changes I made to allow the mutlibooking emails on paid events. I hope someone finds this useful or perhaps suggests something better!
